doc/release/1.2.0-notes.rst

```diff
@@ -6,28 +6,93 @@ SciPy 1.2.0 Release Notes
 
 .. contents::
 
-SciPy 1.2.0 is the culmination of X months of hard work. It contains
-many new features, numerous bug-fixes, improved test coverage and
-better documentation.  There have been a number of deprecations and
-API changes in this release, which are documented below.  All users
-are encouraged to upgrade to this release, as there are a large number
-of bug-fixes and optimizations.  Moreover, our development attention
-will now shift to bug-fix releases on the 1.0.x branch, and on adding
-new features on the master branch.
+SciPy 1.2.0 is the culmination of 6 months of hard work. It contains
+many new features, numerous bug-fixes, improved test coverage and better
+documentation. There have been a number of deprecations and API changes
+in this release, which are documented below. All users are encouraged to
+upgrade to this release, as there are a large number of bug-fixes and
+optimizations. Before upgrading, we recommend that users check that
+their own code does not use deprecated SciPy functionality (to do so,
+run your code with `python -Wd` and check for `DeprecationWarning` s).
+Our development attention will now shift to bug-fix releases on the
+1.2.x branch, and on adding new features on the master branch.
 
-This release requires Python 2.7 or 3.4+ and NumPy 1.8.1 or greater.
+This release requires Python 2.7 or 3.4+ and NumPy 1.8.2 or greater.
 
+For running on PyPy, PyPy3 6.0+ and NumPy 1.15.0 are required.
+
+Release highlights:
+
+   -
```

Comment:
I'd suggest:
- 1-D root finding improvements with a new solver, `toms748`, and a new unified interface, `root_scalar`
- `dual_annealing`
- `shgo`
- (perhaps something else not yet part of the release notes?)